client/stylesheets/_modal.scss

```diff
@@ -371,6 +371,16 @@
 	margin: 8px 0 15px 0 !important;
 }
 
+.profileReceivedPriceOriginal {
+	font-size: 0.8rem;
+	vertical-align: middle;
+	margin-left: 6px;
+}
+
+.profileReceivedPrice b, .profileReceivedPrice span {
```

Comment:
Selector `profileReceivedPrice` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own single line